Makefile

```diff
@@ -41,4 +41,16 @@ lint:
 install:
 	for d in $(SUBDIRS); do $(MAKE) -C $$d install; done
 
+sandbox-test-cri:
+	docker build -f sandbox/cri/Dockerfile -t "cri" .
+	docker run \
```

Comment:
Like https://github.com/firecracker-microvm/firecracker-go-sdk/pull/87#discussion_r265366703, I'd like to see a requirement that this runs as root by checking the UID.